src/test/regress/input/multi_copy.source

```diff
@@ -107,6 +107,15 @@ SELECT count(*) FROM customer_copy_hash;
 -- Confirm that data was copied
 SELECT count(*) FROM customer_copy_hash;
 
+-- Make sure that master_update_shard_statistics() only updates shard length for
```

Comment:
> Do we need to verify append/range tables min/max values are still updated ?


For append tables, in COPY, we call master_update_shard_statistics() to update statistics, so after converting \stage to COPY in regression tests, this will be tested indirectly.

For range tables, it is a bit tricky because we expect the user to set min/max values. I think we should also don't update min/max values on range-partitioned tables? What do you think?